modules/core/src/main/java/org/apache/synapse/config/xml/endpoints/SALoadbalanceEndpointSerializer.java

```diff
@@ -37,6 +37,7 @@
  */
 public class SALoadbalanceEndpointSerializer extends EndpointSerializer {
 
+    private static final String BUILD_MESSAGE = "buildMessage";
```

Comment:
Can't we use the same constant defined in XMLConfigConstants?

Reply:
Done